docs/Board - AlienWii32.md

```diff
@@ -16,11 +16,16 @@ Here are the hardware specifications:
 - motor connections are at the corners for a clean look with reduced wiring
 - dimensions: 29x33mm
 - direct operation from an single cell lipoly battery
-- 3.3V LDO power regulator (ALIENWIIF1 only)
-- 3.3V buck-boost power converter (ALIENWIIF3 only)
-- battery monitoring with an white LED for buzzer functionality (ALIENWIIF3 only)
+- 3.3V LDO power regulator (older prototypes)
+- 3.3V buck-boost power converter (newer prototypes and production versions)
+- battery monitoring with an LED for buzzer functionality (actualy for an ALIENWIIF3 variant)
 
-(*) Spektrum Compatible DSM2 satellites are supported out of the box. DSMX sat will work with DSM2 protocol with default settings (DSM2, 11bit, 11ms is preset). 
+(*) Spektrum Compatible DSM2 satellites are supported out of the box. DSMX sat will work with DSM2 protocol with default settings (DSM2, 11bit, 11ms is preset). This is chosen for maximum compatibility. For optimal connection it is recommended to adjust settings to match the capabilities if you transmitter and satellite receiver. If possible it is recommended to use the DSMX protocol since it is known as more reliable. Also to make use of additional channels you should adjust the following two parameters with the Cleanflight Configurator.
+
+    set serialrx_provider = 1   (0 for 1024bit, 1 for 2048bit) 
+    set spektrum_sat_bind = 5
+    
+For more detail of the different bind modes please refer the �Spektrum bind.md� document
```

Comment:
you can link directory rather than naming the file, naming the file doesn't work for the PDF docs.  See the getting started guide for examples.